runtime/convertValues.go

```diff
@@ -1382,6 +1451,90 @@ func (i valueImporter) importDictionaryValue(
 	), nil
 }
 
+func (i valueImporter) importInclusiveRangeValue(
+	v cadence.InclusiveRange,
+	expectedType sema.Type,
+) (
+	*interpreter.CompositeValue,
+	error,
+) {
+	// start, end and step. The order matters.
+	members := make([]interpreter.IntegerValue, 3)
+
+	var memberType sema.Type
+
+	inclusiveRangeType, ok := expectedType.(*sema.InclusiveRangeType)
+	if ok {
+		memberType = inclusiveRangeType.MemberType
+
+		// The inner type must be an integer.
+		if !memberType.Tag().BelongsTo(sema.IntegerTypeTag) {
+			return nil, errors.NewDefaultUserError(
+				"cannot import inclusiverange: member type must be an integer",
+			)
+		}
+	}
+
+	inter := i.inter
+	locationRange := i.locationRange
+
+	// import members.
+	for index, value := range []cadence.Value{v.Start, v.End, v.Step} {
+		importedValue, err := i.importValue(value, memberType)
+		if err != nil {
+			return nil, err
+		}
+		importedIntegerValue, ok := importedValue.(interpreter.IntegerValue)
+		if !ok {
+			return nil, err
```

Comment:
here it is returning `nil` for both value and the error.

Reply:
Nice catch. Updated now.